npm/private/npm_import.bzl

```diff
@@ -556,6 +557,13 @@ def _impl_links(rctx):
     for dep in ref_deps.keys():
         ref_deps[dep] = ",".join(ref_deps[dep])
 
+    lifecycle_hooks_env_list = rctx.attr.lifecycle_hooks_env
+    lifecycle_hooks_env = {}
+    for env in lifecycle_hooks_env_list:
+        key_value = env.split("=", 1)
+        if len(key_value) == 2:
```

Comment:
should it throw otherwise?

Reply:
@jbedard , sure, I added the `fail` msg here